emails/utils.py

```diff
@@ -41,10 +44,55 @@
 study_logger = logging.getLogger("studymetrics")
 metrics = markus.get_metrics("fx-private-relay")
 
-with open("emails/tracker_lists/level-one-tracker.json", "r") as f:
-    GENERAL_TRACKERS = json.load(f)
-with open("emails/tracker_lists/level-two-tracker.json", "r") as f:
-    STRICT_TRACKERS = json.load(f)
+shavar_prod_lists_url = "https://raw.githubusercontent.com/mozilla-services/shavar-prod-lists/master/disconnect-blacklist.json"
+EMAILS_FOLDER_PATH = pathlib.Path(__file__).parent
+TRACKER_FOLDER_PATH = EMAILS_FOLDER_PATH / "tracker_lists"
+
+
+def get_trackers(level):
+    category = "Email"
+    tracker_list_name = "level-one-trackers"
+    if level == 2:
+        category = "EmailAggressive"
+        tracker_list_name = "level-two-trackers"
+
+    trackers = []
+    file_name = f"{tracker_list_name}.json"
+    try:
+        with open(TRACKER_FOLDER_PATH / file_name, "r") as f:
+            trackers = json.load(f)
+    except FileNotFoundError:
+        trackers = download_trackers(shavar_prod_lists_url, category)
```

Comment:
👍 developer just get trackers as needed, don't need to run the management command

Reply:
Yup! Thanks for recommending that we should download the file like the command does rather than just returning the tracker list from the request.